examples/pod_namespace_watch.py

```diff
@@ -31,6 +31,25 @@ def main():
     v1 = client.CoreV1Api()
     count = 10
     w = watch.Watch()
+    """
+    [INFO: `server-side` & `client-side` timeout]
+
+    `kwargs['timeout_seconds'] = n`:
+       - It is a server-side timeout, i.e. a polite request to the server,
+         mentioning it to cleanly close the connection after n seconds.
+       - If the user have a network outage, then this parameter will do nothing.
+       - The client will hang indefinitely without raising an exception.
+       - You can set this number much higher, maybe to 3600 seconds (1h).
+
+    `kwargs['_request_timeout'] = n`:
+       - It is a client-side timeout, configuring the user's local socket.
+       - If the user have a network outage, resulting into dropping all packets
+         with no RST/FIN, the value of n will specify, for how long
+         the client would wait before realizing the network outage,
+         & dropping the connection.
```

Comment:
https://github.com/kubernetes-client/python-base/pull/36#discussion_r146445193
has some investigations on this, fyi